modules/nw-installing-external-dns-operator-cli.adoc

```diff
@@ -0,0 +1,155 @@
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file that defines the `OperatorGroup` object:
++
+.Example `operatorgroup.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  upgradeStrategy: Default
+    targetNamespaces:
+  - external-dns-operator
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f operatorgroup.yaml
+----
+
+. Create a `Subscription` object:
+
+.. Create a YAML file that defines the `Subscription` object:
++
+.Example `subscription.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  channel: stable-v1
+  installPlanApproval: Automatic
+  name: external-dns-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f subscription.yaml
+----
+
+.Verification
+
+. Get the name of the install plan from the subscription by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get subscription external-dns-operator \
+    --template='{{.status.installplan.name}}{{"\n"}}'
+----
+
+. Verify that the status of the install plan is `Complete` by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get ip <install_plan_name> \
+    --template='{{.status.phase}}{{"\n"}}'
+----
+
+. Verify that the status of the `external-dns-operator` pod is `Running` by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get pod
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE
+external-dns-operator-5584585fd7-5lwqm   2/2     Running   0          11m
+----
+
+. Verify that the catalog source of the subscription is `redhat-operators` by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get subscription
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                    PACKAGE                 SOURCE             CHANNEL
+external-dns-operator   external-dns-operator   redhat-operators   stable-v1
+----
+
+. Check the `external-dns-operator` version by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get csv
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                           DISPLAY                VERSION   REPLACES   PHASE
+external-dns-operator.v<1.y.z>   ExternalDNS Operator   <1.y.z> <1>                Succeeded
+----
+<1>The 1.y.z variable specifies the External DNS Operator version that you installed. Replace 1.y.z with your External DNs Operator version.
```

Comment:
```suggestion
NAME                           DISPLAY                VERSION   REPLACES   PHASE
external-dns-operator.v<1.y.z>   ExternalDNS Operator   <1.y.z>   Succeeded
----
+
The 1.y.z variable specifies the External DNS Operator version that you installed. Replace 1.y.z with your External DNs Operator version.
```

Reply:
Looks like that didn't work afterall. In this case, I wonder if its just the placement of the callout, not being the last item here, that's making it not render 🤔 I think that you could probably get away with no callout here, just add a `+` as I indicate in the suggestion to keep the explanatory text inline.

Reply:
@skrthomas I did away with the explanatory text as it isn't required.